polaris-core/src/main/java/org/apache/polaris/core/persistence/dao/entity/ListEntitiesResult.java

```diff
@@ -46,43 +41,26 @@ public static ListEntitiesResult fromPage(Page<EntityNameLookupRecord> page) {
    * @param errorCode error code, cannot be SUCCESS
    * @param extraInformation extra information
    */
-  public ListEntitiesResult(
-      @Nonnull ReturnStatus errorCode,
-      @Nullable String extraInformation,
-      @Nonnull Optional<PageToken> pageTokenOpt) {
+  public ListEntitiesResult(@Nonnull ReturnStatus errorCode, @Nullable String extraInformation) {
     super(errorCode, extraInformation);
     this.entities = null;
-    this.pageTokenOpt = pageTokenOpt;
   }
 
   /**
    * Constructor for success
    *
    * @param entities list of entities being returned, implies success
    */
-  public ListEntitiesResult(
-      @Nonnull List<EntityNameLookupRecord> entities, @Nonnull Optional<PageToken> pageTokenOpt) {
+  public ListEntitiesResult(Page<EntityNameLookupRecord> entities) {
     super(ReturnStatus.SUCCESS);
     this.entities = entities;
-    this.pageTokenOpt = pageTokenOpt;
   }
 
-  @JsonCreator
-  private ListEntitiesResult(
```

Comment:
ditto the comment on wire-compatibility with previous versions

Reply:
@eric-maynard : It's good that you're able to identify this change as a regression, but what if you're not available ? :) WDYT about encoding this expectation in a unit test?